modules/ROOT/pages/monitor-applications-on-rtf.adoc

```diff
@@ -6,7 +6,7 @@ Monitoring for applications deployed to Runtime Fabric is enabled by default.
 
 Anypoint Monitoring supports the following Mule versions for Runtime Fabric deployments:
 
-* Mule Runtime 3.8.7-AM
+* Mule Runtime 3.8.7
 * Mule Runtime 3.9.1
```

Comment:
We support any 3.9.x version.

Reply:
It might be useful to disambiguate them if a user ctrl-F to find their version? But I am open to condensing that.

Reply:
I suggest using "3.9.x", so we don't need to update this doc every time there is a new runtime patch release

Reply:
Sure thing!